content/en/docs/instrumentation/net/exporters.md

```diff
@@ -149,15 +150,16 @@ services:
 ```csharp
 var builder = WebApplication.CreateBuilder(args);
 
-builder.Services.AddOpenTelemetryTracing(b =>
-{
+builder.Services.AddOpenTelemetry()
+  .WithTracing(b =>
+  {
     b
     .AddZipkinExporter(o =>
     {
         o.Endpoint = new Uri("your-zipkin-uri-here");
     })
     // The rest of your setup code goes here too
-});
+  });
```

Comment:
nit: the indentation used here is 2 spaces rather than 4 like the other code snippets in this doc.

Reply:
fixed